### ISSUE TITLE
Keyerror: images bug

### ISSUE THREAD

**Issue by Pychnight** [d8ahazard/sd_dreambooth_extension#653]
### Kindly read the entire form below and fill it out with the requested information.
```
venv "C:\Users\pychnight\Documents\GitHub\stable-diffusion-webui\venv\Scripts\Python.exe"
Python 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Commit hash: e672cfb07418a1a3130d3bf21c14a0d3819f81fb
Installing requirements for Web UI

#######################################################################################################
Initializing Dreambooth
If submitting an issue on github, please provide the below text for debugging purposes:

Python revision: 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Dreambooth revision: ccd6a73f0a8df9a874f236b35d34a8571296edd2
SD-WebUI revision: e672cfb07418a1a3130d3bf21c14a0d3819f81fb

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.14.dev0 installed.
[+] torch version 1.12.1+cu113 installed.
[+] torchvision version 0.13.1+cu113 installed.

#######################################################################################################

Launching Web UI with arguments: --xformers --autolaunch
```

**Have you read the [Readme](https://github.com/d8ahazard/sd_dreambooth_extension#readme)?**
Yes

**Have you completely restarted the stable-diffusion-webUI, not just reloaded the UI?**
Yes

**Have you updated Dreambooth to the latest revision?**
Yes

**Have you updated the Stable-Diffusion-WebUI to the latest version?**
Yes

**No, really. Please save us both some trouble and update the SD-WebUI and Extension and restart before posting this.
Reply 'OK' Below to acknowledge that you did this.**
OK

**Describe the bug**

starting training fails with keyerror with the lastest dreambooth revision

**Provide logs**

If a crash has occurred, please provide the *entire* stack trace from the log, including the last few log messages *
before* the crash occurred.

raceback (most recent call last):
File "C:\Users\pychnight\Documents\GitHub\stable-diffusion-webui\extensions\sd_dreambooth_extension\scripts\dreambooth.py", line 555, in start_training
result = main(config, use_subdir=use_subdir, lora_model=lora_model_name,
File "C:\Users\pychnight\Documents\GitHub\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\train_dreambooth.py", line 1017, in main
return inner_loop()
File "C:\Users\pychnight\Documents\GitHub\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\memory.py", line 86, in decorator
return function(batch_size, grad_size, *args, **kwargs)
File "C:\Users\pychnight\Documents\GitHub\stable-diffusion-webui\extensions\sd_dreambooth_extension\dreambooth\train_dreambooth.py", line 866, in inner_loop
latents = vae.encode(batch["images"].to(dtype=weight_dtype)).latent_dist.sample()
KeyError: 'images'

**Environment**

What OS?
windows 11

If Windows - WSL or native?
native

What GPU are you using?

NVIDIA 3080 ti, 12 gb

**Screenshots/Config**
If the issue is specific to an error while training, please provide a screenshot of training parameters or the
db_config.json file from /models/dreambooth/MODELNAME/db_config.json

**Comment by Pychnight** [comment]
@d8ahazard well, tried my part to report this issue, hopefully it gets fixed

**Comment by d8ahazard** [comment]
@Pychnight - Sorry about that, was setting up a workflow that wasn't supposed to get to main yet. 

I'm investigating this issue as we speak.

**Comment by d8ahazard** [comment]
There we go. Fixed with e5908ab.
